src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Table(name = "roles")
+@Data
+@NoArgsConstructor
+@Entity
```

Comment:
do not mix an order of annotations 
```suggestion
@Data
@NoArgsConstructor
@Entity
@Table(name = "roles")
```